docker-assets/manageiq.conf

```diff
@@ -1,9 +1,6 @@
 # Timeout: The number of seconds before receives and sends time out.
 Timeout 120
 
-# HTTP Start-up error log
-ErrorLog /var/log/httpd/miq_apache.log
```

Comment:
What about this isn't working?

Also, if we let the errors go to `/var/log` they won't be on the PV.